static/src/javascripts/projects/common/modules/preferences/main.js

```diff
@@ -0,0 +1,69 @@
+define([
+    'react',
```

Comment:
Not yet, but it will be.

I think we should tune for programmer happiness and productivity as well as page render speed, especially on pages that are infrequently accessed, such as preferences and games.

It might also be worthwhile investing more time investigating dead code elimination as opposed to micro libs. We use so much functionality from microlibs now the cumulative size isn't really much different to using jQuery or whatever.

http://swannodette.github.io/2015/01/06/the-false-promise-of-javascript-microlibs/